packages/lodestar-cli/src/cmds/account/cmds/validator/create.ts

```diff
@@ -79,7 +79,10 @@ export async function handler(options: IValidatorCreateOptions): Promise<void> {
   const validatorDirBuilder = new ValidatorDirBuilder(accountPaths);
   const walletManager = new WalletManager(accountPaths);
   const wallet = walletManager.openByName(name);
-  const n = count || atMost - wallet.nextaccount;
+  const n = typeof count === "number" ? count
+    : typeof atMost === "number" ? atMost - wallet.nextaccount
+      : 1;
+  if (isNaN(n)) throw new Error(`Error computing validatorCount: ${n}`);
```

Comment:
Can we throw a YargsError if neither count or atMost are specified?

Reply:
@wemeetagain  what's the rationale for having those fields be required?

Reply:
i know that's what caused this bug, but just wondering why it wasn't required in the first place or how this worked before without it being required (if it worked before)

Reply:
I misread this. It appears its defaulting to count=1 if neither is set.
We don't need them to be required, maybe we can be more clear in comments or cli helptext what the defaults are.
We should check if both are set tho. Error or handle that case.

It worked before if you assumed that either count or atMost were required, and set one of them.

Reply:
@wemeetagain there's already a yargs option configuration `conflicts` that prevents both options from being set at the same time.

I've inherited `atMost` from Lighthouse, tho it's a weird option. Would it make sense to remove `atMost` and just keep `count` defaulting it to 1?

Reply:
:+1: